ReactWindows/ReactNative.Shared/Modules/DevSupport/IDeveloperSettings.cs

```diff
@@ -29,5 +31,7 @@ public interface IDeveloperSettings
         /// Signals whether element inspector is enabled.
         /// </summary>
         bool IsElementInspectorEnabled { get; }
+
+        IPAddress DeviceDebugIpAddress { get; } 
     }
```

Comment:
Let's back this out. I didn't realize sync hooks weren't supported for chrome debugging on other platforms.

I wonder if we should propose that we evaluate sync methods as async methods in case `global.nativeSyncHook` is not installed.